angrmanagement/ui/workspace.py

```diff
@@ -108,6 +137,33 @@ def add_view(self, view, caption, category):
         self.dockable_views.append(dock)
         self.view_to_dockable[view] = dock
 
+    def unsplit_view(self):
+        if self.is_split == 1:
+            id = self.split_tab_id
+            docking_positions = {
+                'left': Qt.LeftDockWidgetArea,
+                'right': Qt.RightDockWidgetArea,
+                'top': Qt.TopDockWidgetArea,
+                'bottom': Qt.BottomDockWidgetArea,
+            }
+        
+            self._main_window.central_widget2.removeDockWidget(self.last_unsplit_view)
+
+
```

Comment:
Nitpick: Too many blank lines. One is enough. Also, please move `unsplit_view()` into the new view manager (PR #73).